### DISCUSSION TITLE
2024/06/04/Qemu.Virt%E5%B9%B3%E5%8F%B0%E9%9B%86%E6%88%90ARM.PL080.DMA/

### DISCUSSION THREAD

**Discussion by giscus** [Jared-ZDC/markel#18] [General]
# 2024/06/04/Qemu.Virt%E5%B9%B3%E5%8F%B0%E9%9B%86%E6%88%90ARM.PL080.DMA/

背景最近有比较多的新员工入职，新员工入职如何更快的适应工作，需要通过一些虚拟项目进行工作培训；但是如果利用现有的芯片平台进行验证，有几个缺陷：  当前芯片略显复杂，新员工接手理解难度较高 平台已经solid，并且有参考的代码以及测试用例，对于新员工而言，没有起到端到端cover一个模块的作用 难以预埋bug，无法通过挖掘设计bug，衡量验证效果  因此根据实际工作需求，设计了一个可以给到新员工练兵

https://markel.top/2024/06/04/Qemu.Virt%E5%B9%B3%E5%8F%B0%E9%9B%86%E6%88%90ARM.PL080.DMA/?t=1722590654056

**Comment by Jared-ZDC** [comment]
尝试评论一下，验证功能
